subsystem/cluster_test.go

```diff
@@ -427,6 +427,49 @@ func installCluster(clusterID strfmt.UUID) *models.Cluster {
 	return c
 }
 
+func tryInstallClusterWithDiskResponses(clusterID strfmt.UUID, successfulHosts, failedHosts []*models.Host) *models.Cluster {
+	Expect(len(failedHosts)).To(BeNumerically(">", 0))
+	ctx := context.Background()
+	reply, err := userBMClient.Installer.InstallCluster(ctx, &installer.InstallClusterParams{ClusterID: clusterID})
+	Expect(err).NotTo(HaveOccurred())
+	c := reply.GetPayload()
+	Expect(*c.Status).Should(Equal(models.ClusterStatusPreparingForInstallation))
+	generateFailedDiskSpeedResponses(ctx, sdbId, failedHosts...)
+	generateSuccessfulDiskSpeedResponses(ctx, sdbId, successfulHosts...)
```

Comment:
By looking on your test I assume the the first diskSpeedResponse is the once to set the way, so why do we need to then `generateSuccessfulDiskSpeedResponses()` ? (looks like we are testing only the failing flow)